search.go

```diff
@@ -68,14 +69,14 @@ type SearchResult struct {
 // search types.  For example, `Search(query, SearchTypeArtist|SearchTypeAlbum)`
 // will search for artists or albums matching the specified keywords.
 //
-// Matching
+// # Matching
```

Comment:
not sure if markdown style comments add any value 🤔

Reply:
My bad on this one. Will remove the extra #s.